action.yml

```diff
@@ -40,8 +63,53 @@ runs:
     - name: Sourcery review
       shell: bash
       run: |
-        if [ "${{ inputs.base_ref }}" == "" ]; then
-          sourcery review ${{ inputs.options }} .
+
+        if [ ${{ inputs.check }} == "true" ]; then
+          check_arg="--check"
+        elif [ ${{ inputs.check }} == "false" ]; then
+          check_arg=""
+        else
+          echo "Invalid value for input 'check'"
```

Comment:
I think these invalid values should fail the script?

Reply:
Done!